lib/orbit/import/helpers.ex

```diff
@@ -0,0 +1,43 @@
+defmodule Orbit.Import.ImportHelpers do
+  @spec parse_csv(Enumerable.t(), [atom()] | boolean(), keyword()) :: Enumerable.t()
+  def parse_csv(file_lines, headers, options \\ []) do
+    file_lines
+    |> Stream.drop(
+      if headers == true do
+        0
+      else
+        1
+      end
+    )
+    |> Stream.map(fn line ->
+      if String.ends_with?(line, "\n") do
+        line
+      else
+        "#{line}\n"
+      end
+    end)
+    |> CSV.decode!(
+      options ++ [headers: headers, field_transform: &String.trim/1, validate_row_length: true]
+    )
+  end
+
+  @doc """
+  Read the csv from the path
+  """
+  @spec import_from_file!(String.t()) :: File.Stream.t()
+  def import_from_file!(file_path) do
+    file_path
+    |> File.stream!()
+  end
+
+  @doc """
+  Read the csv from the `:glides` S3 ref in the given path, and pass it to the provided importer
```

Comment:
nit: What does `` `:glides` `` mean? I think it's a copy-paste from Glides, and might be a reference to how the refs are specified in config, but I don't think it's relevant here.

```suggestion
  Read the csv from the S3 ref at the given path, and pass it to the provided importer
```